### ISSUE TITLE
Failed to merge multiple subtitle images contained in one subPictureBD structure

### ISSUE THREAD

**Issue by hellishfire** [mjuhasz/BDSup2Sub#39]
When converting one bluray sup file recently, I noticed that the ouput idx/sub file contains incomplete subtitles for certain lines.

After single-stepping through the source code, I found out that when decoding images, only the first imageObject in a subPictureBD instance was decoded. This led to incomplete subtitle since in my case, there were sometimes two imageObject in one subPictureBD. I believe these two imageObject should be combined to form one complete subtitle image.

I'm not a Java programmer, so I suppose I will just post this issue here instead of trying to fix this bug myself. I can provide the sup file in question if further inspection is needed.

**Comment by amichaelt** [comment]
Yes, that is the problem with the code.  It always only assumed one image object.  It's the same reason the avatar bluray SUP from: https://github.com/mjuhasz/BDSup2Sub/issues/34 crashes the program.  I'm sure another SUP file to help fix the issue and test would be helpful.

**Comment by ghost** [comment]
Miklos, another good test file for multi-region subtitles is here: https://www.dropbox.com/s/yi08oqi3jtbfrdl/Ouran1a.7z That exposed a number of bugs in the multi-region parsing support I added.

**Comment by mjuhasz** [comment]
Got it, thank you! I am quite busy these days but I certainly downloaded your file and will look into it later.

**Comment by xwlcn** [comment]
It seems that the author has not maintained for a long time, and the issue has not been fixed.

So I finished this work, You can be found here: [https://github.com/xwlcn/BDSupSubPlus](https://github.com/xwlcn/BDSupSubPlus)
